app/src/main/res/layout/activity_login.xml

```diff
@@ -30,4 +30,13 @@
         android:layout_alignParentStart="false"
         android:layout_centerInParent="true" />
 
+    <Button
+        android:id="@+id/button2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Import Database"
```

Comment:
Should this text be a string resource? I know it's for testing purposes but still.